Samples/OpenQasmReader.Tests/Validate/Hadamard.qs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.Quantum.Samples.OpenQasmReader.Tests.Validate
+{
+ open Microsoft.Quantum.Primitive;
+ open Microsoft.Quantum.Canon;
+ open Microsoft.Quantum.Extensions.Math;
+
+ operation HadamardTest():(Result[])
+ {
+  body
+  {
+   mutable c = new Result[1];
+   using(q = Qubit[1]){
+   H(q[0]);
+   set c[1] = M(q[0]);
+   ResetAll(q);
```

Comment:
why the weird indentation of this one?

Reply:
Same indentation as the rest. Is caused because the using indent is not added to the rest yet. Added some logic to make it a bit better.